sdk/resourcemanager/azure-resourcemanager-storage/src/test/java/com/azure/resourcemanager/storage/StorageAccountOperationsTests.java

```diff
@@ -219,14 +219,9 @@ public void canDisallowCrossTenantReplicationOnStorageAccount() {
                 .withRegion(Region.US_EAST2)
                 .withNewResourceGroup(rgName)
                 .withSku(StorageAccountSkuType.STANDARD_LRS)
+                .allowCrossTenantReplication()
                 .create();
 
-        Assertions.assertFalse(storageAccount.isAllowCrossTenantReplication());
-
-        storageAccount.update()
-            .allowCrossTenantReplication()
-            .apply();
```

Comment:
keep the update test, just change to disallow

Reply:
The `allowCrossTenantReplication` interface has been added in `Create` mode, and this interface has been called in the test case `canDisallowCrossTenantReplicationOnStorageAccount`,
```
        StorageAccount storageAccount =
            storageManager
                .storageAccounts()
                .define(saName)
                .withRegion(Region.US_EAST2)
                .withNewResourceGroup(rgName)
                .withSku(StorageAccountSkuType.STANDARD_LRS)
                .allowCrossTenantReplication()
                .create();
```
 so 
```
        storageAccount.update()
            .allowCrossTenantReplication()
            .apply();
``` 
can be removed. However, verification of the initial setting value of `Cross-Tenant Replication` is still necessary.

Reply:
Just add a 

```java
        storageAccount.update()
            .disallowCrossTenantReplication()
            .apply();
```

Reply:
Removed the invoke of interface `disallowCrossTenantReplication` from the test case `canAllowCrossTenantReplicationOnStorageAccount`.